src/main/java/com/zufar/onlinestore/cart/entity/ShoppingSession.java

```diff
@@ -9,16 +9,14 @@
 import jakarta.persistence.Id;
 import jakarta.persistence.OneToMany;
 import jakarta.persistence.Table;
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
+import lombok.*;
 
 import java.time.LocalDateTime;
 import java.util.Objects;
 import java.util.Set;
 import java.util.UUID;
 
+@Builder
 @Getter
 @Setter
```

Comment:
pls use fetch type lazy or provide some clarity why you didn't choose this strategy

Reply:
add //TODO for this comment